frontend/src/views/projectEdit.js

```diff
@@ -208,6 +209,10 @@ export function ProjectEdit({ id }) {
       filtered.push({ locale: null, fields: ['mappingTypes'] });
     }
 
+    if (!projectInfo.organisation) {
```

Comment:
We could improve that to avoid repeated messages:

![image](https://user-images.githubusercontent.com/666291/104454097-19b0e700-5584-11eb-8356-451c35287b81.png)

Reply:
<img width="409" alt="Screenshot 2021-01-18 at 18 10 42" src="https://user-images.githubusercontent.com/31903212/104932315-8b3fc980-59b8-11eb-91a3-6966238394a6.png">